tests/Api.IntegrationTests/Endpoints/Magic/MagicTests.cs

```diff
@@ -74,7 +74,7 @@ public async Task I_receive_a_validation_error_when_the_url_does_not_contain_the
         responseDetails.Should().NotBeNull();
         var magicLinkUrlError = responseDetails!.Errors.FirstOrDefault(x => x.Key.Equals(nameof(request.UrlTemplate), StringComparison.CurrentCultureIgnoreCase));
         magicLinkUrlError.Should().NotBeNull();
-        magicLinkUrlError.Value.Should().ContainMatch("Value must contain the `<token>` template.");
+        magicLinkUrlError.Value.Should().ContainMatch($"You have provided a {nameof(request.UrlTemplate)} without a <token> template. For example, https://www.exmaple.com?token=<token>");
```

Comment:
Should probably be just `Should().Contain()` not `Should().ContainMatch()` (there is no regex)